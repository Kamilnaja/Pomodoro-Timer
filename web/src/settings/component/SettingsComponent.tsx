import React from 'react';
import { Button, Container, Form } from 'react-bootstrap';
import './settingsComponent.scss';

export const SettingsComponent = () => {
  return (
    <Container className="settings">
      <Form>
        <Form.Group controlId="formBasicCheckbox">
          <Form.Check type="checkbox" label="Enable sounds" />
        </Form.Group>
        <Button variant="primary" type="submit">
          Submit
        </Button>
      </Form>
    </Container>
  );
};
