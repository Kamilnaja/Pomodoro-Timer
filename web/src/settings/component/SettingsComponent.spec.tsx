import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { SettingsComponent } from './SettingsComponent';

describe('SettingsComponent', () => {
  const component = shallow(<SettingsComponent />);

  it('Should match snapshot', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
