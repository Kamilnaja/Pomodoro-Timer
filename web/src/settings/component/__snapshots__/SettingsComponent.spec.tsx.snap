// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsComponent Should match snapshot 1`] = `
<div>
  Settings - work in progress
</div>
`;
